app/views/components/docs/document-list.yml

```diff
@@ -0,0 +1,43 @@
+name: Document list
+description: A list of links to documents including document type and when updated.
+body: |
+  The component expects data in a specific format from a content item.
+
+  If a group_index parameter is included the component adds link tracking attributes to each link.
+acceptance_criteria: |
+  The component must:
+
+  * inform the user how many items are in the list
+
+  Links in the component must:
+
+  * accept focus
+  * be focusable with a keyboard
+  * be usable with a keyboard
+  * indicate when it has focus
```

Comment:
I think we need to have a wider discussion about some of Leonie's suggestions. There's a danger we may end up over-specifying these criteria. Most of these are about links, so we could just define criteria for links and say they apply to all links unless otherwise specified.